### PR TITLE
Move host gradient underneath audio icon

### PR DESCRIPTION
**Pull request by kajsaunge**
Issue: [Host spotlight has mic behind the gradient](https://www.notion.so/29k/Early-Access-2794500652b34c64b0aff0dbbc53e0ab#2535457caab845d2b0a7e600e74f1239)

### Description of the Change
- Move spotlight gradient from Host to Participant component

**Before**
| iOS | Android |
|-|-|
| image | image |
**After**
| iOS | Android |
|-|-|
| image | image |

### CODE REVIEW

**Review by gewfy — APPROVED** [review]
👌

**Comment by kajsaunge** [comment]
@gewfy I removed the Host component now, that's fine right? I mean it seems redundant but Im unsure if not passing an active state will mess things up? Works fine locally 😃

File: client/src/routes/Session/components/Slides/Slide.tsx [hunk 1]
```diff
@@ -35,7 +37,9 @@ export const Slide = React.memo(({slide, active}: SlideProps) => {
 
   return (
     <Wrapper backgroundColor={theme?.backgroundColor}>
-      {slide.type === 'host' && <Host active={active} />}
+      {slide.type === 'host' && (
+        <Participant participant={participantSpotlight} isHostSpotlight />
+      )}
```
Comment (gewfy) [inline client/src/routes/Session/components/Slides/Slide.tsx:42]: I'd like to keep the `Host` slide abstraction but simplify it to this?

File: client/src/routes/Session/components/Participants/Participant.tsx [hunk 2]
```diff
@@ -70,13 +73,27 @@ const Heading = styled(Display36)({
...
+  position: 'absolute',
+  left: 0,
+  right: 0,
+  top: 0,
+  height: 80,
+});
+
 type ParticipantProps = {
   participant: DailyParticipant;
+  isHostSpotlight?: boolean;
```
Comment (gewfy) [inline client/src/routes/Session/components/Participants/Participant.tsx:86]: Nit pick, but could we rename this to `topGradient` instead?

File: client/src/routes/Session/components/Slides/Slide.tsx (hunk 1 above)
Comment (kajsaunge) [reply client/src/routes/Session/components/Slides/Slide.tsx:42]: Interested in the reasoning but let's keep it!

**Pull request merged**